src/coreclr/jit/emitarm64.cpp

```diff
@@ -1904,6 +1904,14 @@ static const char * const  pRegNames[] =
     "p10", "p11", "p12", "p13", "p14",
     "p15"
 };
+
+static const char * const  pnRegNames[] =
```

Comment:
good catch. Seems this was missed in https://github.com/dotnet/runtime/pull/97068